modules/kubernetes/starboard/templates/starboard-values.yaml.tpl

```diff
@@ -10,7 +10,10 @@ trivy:
   mode: ClientServer
   severity: MEDIUM,HIGH,CRITICAL
   ignoreUnfixed: true
-  serverURL: "http://trivy.trivy.svc.cluster.local:4954"
+  serverURL: "http://trivy.starboard-operator.svc.cluster.local:4954"
+  # The trivy image have to match with the trivy server
+  imageRef: docker.io/aquasec/trivy:0.23.0
```

Comment:
Is this going to cause problem when automating version updates?

Reply:
Running terraform without any manual job you mean?
Since we are deleting and patching helm charts both for trivy and starboard I don't think this should be a problem.

Reply:
Removed it since currently it's not a problem to run 0.23.0 together with trivy 0.24.0.
We have to look at the changelog allot to make sure that there have been no breaking changes that might create issues for us.